notebooks/01_Introduction/01_02_TaylorExpansion.md

```diff
@@ -491,7 +491,7 @@ A nice feature of matplotlib is that it can display rendered LaTeX equations in
 
 ## Summary
 
-In this notebook we have introduced the concepts of numerical precision and numerical error. We also discussed different approaches to build numerical sequences in Python and learned about NumPy arrays - fundamental objects for scientific programming in Python. Using the Matplotlib package, we gave a trivial overview on 2D visualization and illustrated that the error term in Taylor series is of order $\mathcal{O}(\Delta x)^{n+1}$ when the last expansion term considered is of order $\mathcal{O}(\Delta x)^{n}$.
+In this notebook we have introduced the concepts of numerical precision and numerical error. We also discussed different approaches to build numerical sequences in Python and learned about NumPy arrays - fundamental objects for scientific programming in Python. Using the Matplotlib package, we gave a trivial overview on 2D visualization and illustrated that the error term in Taylor series is of order $\mathcal{O}(\Delta x^{n+1})$ when the last expansion term considered is of order $\mathcal{O}(\Delta x^{n})$.
```

Comment:
I don't see, what has changed here, but it's marked as a change. Can you explain?